Assets/Scripts/DayCell.cs

```diff
@@ -81,6 +109,61 @@ private bool SetImageActiveDependancy(Image image)
     }
 
 
+    private void TryRemoveCellData()
+    {
+        if (!daycellData.isPoopImageActive && !daycellData.isMedicineImageActive && daycellData.description == "")
+        {
+            StateSaver.Data.Remove(DaycellData);
+        }
+    }
+
+
+
+    public void Select()
+    {
+        if (selectImage.enabled == false)
+        {
+
+            SelectCellImage(this);
```

Comment:
could these be renamed to be a bit clearer? I read SelectCell and CellSelect and my brain gets confused? Minor thing no worries if you dont think its appropriate

Reply:
Resolved